index.html

```diff
@@ -20,10 +20,306 @@
+                  <li class="rating--bubble"><blockquote>"Jenna Ortega shines as the oldest Addams' daughter, it's almost as if she was born to play the child of woe."</blockquote></li>                
+                  <li class="rating--bubble"><blockquote>" Wednesday might be one of the greatest shows I have ever watched. It is incredibly amazing."</blockquote></li>
+              </ul>
+            </div>
+            <!--Button to see more ratings/leave users' own rating/chat with others-->
+            <div class="ratings--bottom">
+              <p class="ratings--info">Leave your own review, chat with others, and more</p>
+              <button class="black--button">See more</button>
+            </div>
+          </div>
+        </section>
+    <!--End of Content Section: Ratings-->
+
+    <!--Content Section: Watch-->
+        <section>
+          <h2 class="sr-only">Section explains that you can access Wednesday and more Addams family content from the cite</h2>
+            <div class="watch--section">
+              <h2 class= "White--title">Watch Wednesday and more...</h2>  
+              <!--Poster Images. Set into an unordered list-->          
+                <div class="Posters">
+                  <ul>
+                    <li>
+                      <img 
+                      class="Media--poster"
+                      src="./images/Addams-Family-1964.png" alt="Addams Family 1964 Poster">
+                    </li>
+                    <li>
+                      <img 
+                      class="Media--poster"
+                      src="./images/Wednesday-Poster.png" alt="Wednesday Poster">
+                    </li>
+                    <li>
+                      <img 
+                      class="Media--poster"
+                      src="./images/Addams-Family-1991.png" alt="Addams Family">
+                    </li>
+                  </ul>
+                </div> 
+                <!--Button to sign into/sign up for Netflix-->
+                <div class="watch--bottom"> 
+                    <button class="white--button">View all</button>
+                </div>
+            </div>
+        </section>
+      <!--End of content section: watch-->
+
+    <!--Content Section: Behind the Scenes-->    
+        <section>
+          <h2 class="sr-only">Behind the scenes section</h2>
+            <div class="bts-section">
+              <div class="bts-text">
+                  <h2 class="bts--title">Go behind the scenes</h2>
+                  <p class="subtitle">Learn about the team, crew, actors and all their hard work that brought Wednesday to death</p>
+                  <div class="center--button--alignment">
+                    <button class="white--button">See more</button>
+                  </div>
+              </div>
```

Comment:
If these "buttons" are links, I would recommend they be marked up as `<a href="#">See more</a>`

"Buttons" are generally reserved for interactive page elements - like our "Add To Do" button in our To Do list app for example. 

Links should be marked up using an `<a href="#"...`